src/es6/Entity.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-import { entityFactory, dispose } from './EntityFactory.js';
+import { entityFactory, dispose } from './EntityFactory.js.js';
```

Comment:
Guessing this change was accidental?

Reply:
Yes, I believe he meant `./EntityFactory.js.js.js`

Reply:
yeah, probably just vscode trying to auto reference change